### PR TITLE
Replaces the IAA with the Representative job

### PR DESCRIPTION
**Pull request by Alberyk**
This pr does what is suggested in this thread: https://forums.aurorastation.org/topic/12395-iaa-rework-representatives/

### CODE REVIEW

**Comment by Bygonehero** [comment]
Hopefully more representatives are added in the future, otherwise, this is exchanging a job that many species can enjoy to one only human and tajara can.

**Comment by fernerr** [comment]
The back door via security is still named internal affairs, could think about removing it wholesale though.
If you do, perhaps the powerline should be redrawn so the office draws power from the bridge grid rather than the security grid.

**Comment by fernerr** [comment]
Rather than hooking up to the mains I meant like so 
wire

File: code/game/jobs/faction/nanotrasen.dm [hunk 1]
```diff
@@ -12,3 +12,21 @@
...
+	var/objective
+	switch(mission_level)
+		if("high")
+			objective = pick("Have [rand(1,4)] crewmember sign NT apprenticeship contracts",
+						"Make sure that the station fullfils [rand(4,12)] cargo bounties",
+						"Make sure that the station raises [rand(5000,12000)] credits by the end of the shift")
+		if("medium")
+			objective = pick("Have [rand(2,5)] crewmembers sign contract extensions",
+						"Have [rand(2,5)] crewmembers buy Odin real estate",
+						"[rand(3,10)] crewmember must buy Getmore products from the vendedors")
```
Comment (fernerr) [inline code/game/jobs/faction/nanotrasen.dm]:
```suggestion
						"[rand(3,10)] crewmember must buy Getmore products from the vendors")
```

File: code/game/jobs/faction/nanotrasen.dm [hunk 2]
```diff
@@ -12,3 +12,21 @@
...
+		else
+			objective = pick("Conduct and present a survey on crew morale and content",
+						"Make sure that [rand(2,4)] complaints are solved on the station",
+						"Have [rand(3,10)] crewmembers buy Getmore products from the vendedors")
```
Comment (fernerr) [inline code/game/jobs/faction/nanotrasen.dm]:
```suggestion
						"Have [rand(3,10)] crewmembers buy Getmore products from the vendors")
```

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -87,6 +87,16 @@
 	desc = "A stylish, if purple, beret."
 	icon_state = "purpleberet"
 
+/obj/item/clothing/head/beret/liaison
+	name = "corporate liaison beret"
+	desc = "A stylish worn by corporate liaisons."
```
Comment (fernerr) [inline code/modules/clothing/head/jobs.dm]:
```suggestion
	desc = "A stylish beret worn by corporate liaisons."
```

File: code/modules/clothing/head/jobs.dm [hunk 4]
```diff
@@ -87,6 +87,16 @@
 	desc = "A stylish, if purple, beret."
 	icon_state = "purpleberet"
 
+/obj/item/clothing/head/beret/liaison
+	name = "corporate liaison beret"
+	desc = "A stylish worn by corporate liaisons."
+	icon_state = "centcomofficerberet"
+
+/obj/item/clothing/head/beret/centcom/officer
```
Comment (fernerr) [inline code/modules/clothing/head/jobs.dm]: This is a duplicate of the beret below.

**Pull request merged**